Rollbar.NetPlatformExtensions/RollbarLogger.cs

```diff
@@ -203,52 +202,44 @@ mslogging.EventId eventId
 
         private bool disposedValue = false; // To detect redundant calls
 
+
         /// <summary>
         /// Releases unmanaged and - optionally - managed resources.
         /// </summary>
         /// <param name="disposing">
         ///   <c>true</c> to release both managed and unmanaged resources; 
         ///   <c>false</c> to release only unmanaged resources.
         /// </param>
+        [System.Diagnostics.CodeAnalysis.SuppressMessage("Major Code Smell", "S1066:Collapsible \"if\" statements should be merged", Justification = "Cleaner code structure.")]
         protected virtual void Dispose(bool disposing)
         {
             if (!disposedValue)
             {
                 if (disposing)
                 {
-                    // TODO: dispose managed state (managed objects).
+                    // dispose managed state (managed objects).
```

Comment:
It's not clear what is happening here and similar places in the code. Was the TODO stale? Or is it still a TODO, but the lint complained about the comment structure?

Reply:
@waltjones , thanks for reviewing this!
Most of these TODOs are generated from code templates generated by Visual Studio. For example, when it templates standardized Disposable pattern implementation. So, these TODOs are reminders/placeholders to fill in as applicable. I, in general, tend to keep around even the ones that are not applicable at the moment since in the future a class implementing the pattern, for example,  may gain access to some non-managed resources and will have to fill in some resources cleanup code near these TODOs. On the other hand, SonarCloud looks for the TODO markers and complains about these. 
As the solution to this, I decided to just remove the TODO markers but to keep their attached comments in place as reminders but without the markers that upset the SonarCloud analyzer.

Reply:
I guess my concern is that they now look like declarative statements of what's already implemented. As time goes on, how do you (or other implementors and reveiwers) keep up with what's really a TODO or not?

This is a situation where I wonder if it is better to globally disable the lint rule.

Reply:
I am hesitating to disable the rule globally since we do occasionally have valid TODOs to implement later on.
BTW, to me, these comments sound more like directions to follow rather than a "done deal".